Microsoft.Toolkit.Uwp.UI.Controls/RemoteDevicePicker/RemoteDevicePicker.cs

```diff
@@ -0,0 +1,358 @@
+            }
+
+            var statusType = typeof(RemoteSystemStatusType).GetEnumNames().OrderBy(a => a.ToString()).ToList();
+            _deviceStatus.ItemsSource = statusType;
+
+            if (_statusFilter != null)
+            {
+                _deviceStatus.SelectedValue = _statusFilter.RemoteSystemStatusType.ToString();
+            }
+
+            var authType = typeof(RemoteSystemAuthorizationKind).GetEnumNames().OrderBy(a => a.ToString()).ToList();
+            _authorizationType.ItemsSource = authType;
+
+            if (_authorizationKindFilter != null)
+            {
+                _authorizationType.SelectedValue = _authorizationKindFilter.RemoteSystemAuthorizationKind.ToString();
+            }
+        }
+
+        private void BuildFilters(RemoteSystemDiscoveryTypeFilter discoveryFilter, RemoteSystemAuthorizationKindFilter authorizationKindFilter, RemoteSystemStatusTypeFilter statusFilter)
+        {
+            var filters = new List<IRemoteSystemFilter>();
+            if (discoveryFilter != null)
+            {
+                filters.Add(discoveryFilter);
+            }
+
+            if (authorizationKindFilter != null)
+            {
+                filters.Add(authorizationKindFilter);
+            }
+
+            if (statusFilter != null)
+            {
+                filters.Add(statusFilter);
+            }
+
+            var remoteDeviceHelper = new RemoteDeviceHelper(filters);
+            RemoteSystems = remoteDeviceHelper.RemoteSystems;
+
+            UpdateProgressRing(true);
+            UpdateList();
+        }
+
+        private void ListDevices_ContainerContentChanging(ListViewBase sender, ContainerContentChangingEventArgs args)
+        {
+            var model = (RemoteSystem)args.Item;
+            switch (model.Status)
+            {
+                case RemoteSystemStatus.Available:
+                    args.ItemContainer.IsEnabled = true;
+                    break;
+
+                case RemoteSystemStatus.DiscoveringAvailability:
+                case RemoteSystemStatus.Unavailable:
+                case RemoteSystemStatus.Unknown:
+                    args.ItemContainer.IsEnabled = false;
+                    break;
+            }
+        }
+
+        private void ListDevices_SelectionChanged(object sender, SelectionChangedEventArgs e)
+        {
+            if (_listDevices.SelectionMode == ListViewSelectionMode.Single)
+            {
+                ReturnDevices();
+                Hide();
+            }
+            else
+            {
+                if (_listDevices.SelectedItems.Count > 0)
+                {
+                    IsSecondaryButtonEnabled = true;
+                }
+                else
+                {
+                    IsSecondaryButtonEnabled = false;
+                }
+            }
+        }
+
+        private void ListDevices_DoubleTapped(object sender, DoubleTappedRoutedEventArgs e)
+        {
```

Comment:
You are absolutely correct. I propose to enable Double Tap only for Single Selection Mode. That way the confusion with multiple mode can be eliminated. and for single mode, it think the current way is correct because if you are double tapping on an item, you are try to select an item even though it is disabled and cannot be used.

Reply:
It was [this](https://github.com/windows-toolkit/WindowsCommunityToolkit/pull/2126#issuecomment-406105459) comment for which i have enabled double tap on the list. Better suggestion would be to remove the double tap completely to avoid confusion. ping @JustinXinLiu for his comment.

Reply:
Yeah, Justin's offline at the moment, we can always add double-tap behavior later.  However, your latest change has the inverted logic (oops).